pandas/core/resample.py

```diff
@@ -519,21 +519,55 @@ def nearest(self, limit=None):
 
     def backfill(self, limit=None):
         """
-        Backward fill the values
+        Backward fill the values.
+
```

Comment:
It might be interesting to add a more thorough explanation of what exactly is a backward fill for novice users who have never seem this term. Something along the lines of: 'get all the NA values and substitute with the value on the next row that has a non-NA value'

Reply:
if you can find a wikipedia reference would be great as well.